app/components/members/delete_dialog_component.html.erb

```diff
@@ -40,7 +40,7 @@ See COPYRIGHT and LICENSE files for more details.
     <% dialog.with_footer do %>
       <%= render(Primer::Beta::Button.new(data: { close_dialog_id: id })) { "Cancel" } %>
       <%= render(Primer::Beta::Button.new(scheme: :danger, tag: :a, href: delete_url(delete_member: true), data: { method: :delete })) { "Keep only shared" } %>
-      <%= render(Primer::Beta::Button.new(scheme: :danger, tag: :a, href: delete_url(delete_member: true, delete_shared_work_packages: true), data: { method: :delete })) { "Remove all" } %>
+      <%= render(Primer::Beta::Button.new(scheme: :danger, tag: :a, href: delete_url(delete_member: true, delete_work_package_shares: true), data: { method: :delete })) { "Remove all" } %>
```

Comment:
Thanks, this is no longer misleading.

Reply:
Noted while fixing specs that this reads wrong